javascript.ru/2.12.2/2.12.2.js

```diff
@@ -0,0 +1,9 @@
+﻿var value = prompt('Are the unicorns real', '');
+
+if (value.toUpperCase() == 'Yes'.toUpperCase() ) {
+	alert('right!');
+}
+
+else {
+	alert(' It is not right. Of course they are real!');
```

Comment:
`' It is not right...` =>`'It is not right...` (remove space).

Reply:
Done cbe8a98.